add: Onchain gaming page (#15453)
ethereum/ethereum-org-website: public/content/gaming/index.md

---
title: Onchain Gaming
lang: en
template: use-cases
image: /images/robot-help-bar.png
sidebarDepth: 2
summaryPoint1: Game rules and state can be enforced by blockchain, not a studio’s servers
summaryPoint2: Anyone can build mods, bots or entirely new games that plug into the same on‑chain data

> summaryPoint3: Purpose‑built L2s such as Redstone and frameworks like MUD cut costs enough to support real‑time gameplay

Reviewer: ```suggestion
summaryPoint3: Purpose‑built L2s, like Redstone, and frameworks, like MUD, cut costs enough to support real‑time gameplay
```

buttons:
  - content: Learn more
    toId: how-gaming-on-ethereum-works
  - content: Explore apps
    toId: popular-games-built-on-ethereum
    isSecondary: false
---

## How gaming on Ethereum works {#how-gaming-on-ethereum-works}

Gaming on Ethereum comes in various forms, from games that integrate blockchain for specific features to those where the entire game world lives onchain. Many games utilize Ethereum to manage in-game assets as NFTs (Non-Fungible Tokens). This allows players to truly own unique digital items, which can be openly traded, sold, or gifted outside the confines of a single game developer's ecosystem. While these assets offer new forms of player agency, the core game logic often remains on centralized servers.

> True Onchain Games are games where the fundamental mechanics and often the entire game world are directly governed by smart contracts on the Ethereum blockchain (or its Layer 2s). This ensures unparalleled transparency. No central servers, no intermediaries—just transparent, player‑driven experiences and economy. 

Reviewer: ```suggestion
Purely onchain games are games where the fundamental mechanics, and often the entire game world, are directly governed by smart contracts on the Ethereum blockchain (or its Layer 2s). This ensures unparalleled transparency. No central servers, no intermediaries—just transparent, player‑driven experiences and economies. 
```

- Players own their assets as NFTs.
- Items can be freely traded, gifted, or sold.
- Blockchain ensures assets remain accessible forever.

## The current state of gaming {#the-current-state-of-gaming}

- **Frequent Game Shutdowns:** In 2023 alone, [60+ games were shut down](https://tech4gamers.com/game-studios-shut-down-2023/), and 11 game studios closed entirely, leaving players with nothing to show for their in‑game investments. Onchain games, with their logic and assets on a decentralized network, can persist as long as the blockchain exists, offering a higher degree of permanence.
- **Frustration Over Locked Assets:** [51% of gamers feel frustrated](https://www.starknet.io/blog/blockchain-gaming/) that they can’t gift or re‑sell in‑game items they buy, and 23% are annoyed by how difficult it is to recover money from in‑game purchases. Players invest significant time and money into acquiring in-game items, only to find they don't truly own them. Ethereum's NFT standard provides verifiable digital ownership, ensuring players control their assets.

> - **High Spending with No Return:** [Gamers spend an average of \$6,425](https://www.starknet.io/blog/blockchain-gaming/) on virtual items over their lifetime, with \$8.74 spent per month or \$104 per year.

Reviewer: ```suggestion
- **High Spending with No Return:** [Gamers spend an average of \$6,425](https://www.starknet.io/blog/blockchain-gaming/) on virtual items over their lifetime, with \$8.74 spent per month or \$104 per year. Onchain ownership transforms that spending from a sunk cost into an investment in a  digital asset that can be sold, traded, or gifted, much like a physical collectible. 
```

## Popular games built on Ethereum {#popular-games-built-on-ethereum}

Developers are exploring new ways to make games more engaging and go beyond simple reward mechanics to deepen, skill‑based gameplay.

<OnchainGamingProductList list="game" />

## Play-to-earn (P2E) {#play-to-earn-p2e}

> With Play‑to‑Earn (P2E) games, you can earn assets with real value. Unlike early P2E models that relied on unsustainable rewards, newer games focus on long‑term value. For example, [Wolf Game](https://gam3s.gg/wolf-game/) combines strategic gameplay with real asset ownership. Players manage virtual sheep and wolves, earning the in‑game currency WOOL, which can be traded or sold.

Reviewer: ```suggestion
With Play‑to‑Earn (P2E) games, you can earn in-game assets with real-world value. Unlike early P2E models that relied on unsustainable rewards, newer games focus on long‑term value. For example, [Wolf Game](https://gam3s.gg/wolf-game/) combines strategic gameplay with real digital asset ownership. Players manage virtual sheep and wolves, earning the in‑game currency WOOL, which can be traded or sold.
```

<OnchainGamingProductList list="p2e" />

## Interoperability & cross‑chain play {#interoperability-and-cross-chain-play}

> One of Ethereum's most powerful features for gaming is its native support for interoperability and composability. Unlike walled-garden traditional games, assets and even game logic built on Ethereum can potentially interact across different applications and chains without sacrificing security. While this is still in development, some game networks are already interoperable, where in‑game items (NFTs) can be used across multiple games.

Reviewer: ```suggestion
One of Ethereum's most powerful features for gaming is its native support for interoperability and composability. Traditional games operate in "walled-gardens," locking in-game assets and progress to a single title. In-game assets, and even core game logic, built on Ethereum can potentially interact across different, games, applications, and chains, without sacrificing security. While this is still in development, some game networks are already interoperable, where in‑game items (NFTs) can be used across multiple games.
```

For instance, in Illuvium, [players can collect creatures called Illuvials](https://gam3s.gg/news/illuvium-three-web3-games/), which are NFTs. These Illuvials can be used across various games within the Illuvium universe. An Illuvial captured in Illuvium Overworld can also be used in Illuvium Arena for battles.

Another example is Galaxy Fight Club. In this game, [players can use different NFT collections](https://gam3s.gg/galaxy-fight-club/) to participate in battles, meaning NFTs from various projects can be used within the game.

## Scalability & gas fee improvements {#scalability-and-gas-fee-improvements}

> As Ethereum has matured, solutions exist to significantly reduce the cost of onchain gaming and enhance performance, making interactive and fast-paced experiences more viable. These advancements are primarily achieved through:

Reviewer: ```suggestion
An old critique of onchain gaming was that blockchains are too burdensome and slow to provide gamers with the experience they expect. But as Ethereum has matured, solutions to significantly reduce the cost and enhance the performance of onchain gaming have emerged, making interactive and fast-paced experiences viable. 

These advancements are primarily achieved through:
```

> - **Gas-less Transactions:** some platforms and protocols, including those leveraging Immutable X, offer the ability to conduct transactions like trading or minting NFTs without fees for the user, further streamlining the player experience

Reviewer: ```suggestion
- **Gas-less Transactions:** some platforms and protocols, including those leveraging Immutable X, offer the ability to conduct transactions like trading or minting NFTs without fees for the user, further streamlining the player experience.
```

> - **Scaling Solutions:** Arbitrum, zkSync and Starknet are actively fostering ecosystems for onchain games due to their high throughput and low costs.

Reviewer: ```suggestion
- **Scaling Solutions:** Ethereum L2s like Arbitrum, zkSync and Starknet are actively fostering ecosystems for onchain games due to their high throughput and low costs.
```
